services/analytics-service.ts

```diff
@@ -34,6 +40,11 @@ export class AnalyticsService implements IAnalyticsService {
 
 					let trackFeatureUsage = this.$prompter.confirm(message, () => true).wait();
 					this.setStatus(this.$staticConfig.TRACK_FEATURE_USAGE_SETTING_NAME, trackFeatureUsage).wait();
+					if(trackFeatureUsage) {
+						this.trackFeatureCore(`${this.acceptTrackFeatureSetting}.true`).wait();
+					} else {
+						this.trackFeatureCore(`${this.acceptTrackFeatureSetting}.false`).wait();
+					}
```

Comment:
You can simplify `if else` with the following
`this.trackFeatureCore(`${this.acceptTrackFeatureSetting}.${!!trackFeatureUsage}`).wait();`